xls/modules/fpsub_2x32.x

```diff
@@ -20,31 +20,28 @@
 // In all other cases, results should be identical to other
 // conforming implementations (modulo exact significand
 // values in the NaN case.
-import float32
+import xls.modules.apfloat_sub_2
 import xls.modules.fpadd_2x32
+import float32
 
 type F32 = float32::F32;
 
 pub fn fpsub_2x32(x: F32, y: F32) -> F32 {
-  let y = F32 {sign: !y.sign, ..y};
-  fpadd_2x32::fpadd_2x32(x, y)
+  apfloat_sub_2::apfloat_sub_2<u32:8, u32:23>(x,y)
```

Comment:
Same comment as below (aggregate stuff in a single file, update names, comments, etc.)

Reply:
See above.